### PR TITLE
Improve hidden Notifications page

### PR DESCRIPTION
**Pull request by beevee**
Note that there is a lot of whitespace changes in `NotificationList.less`, since it was formatted with Windows-style line endings.

Fix moira-alert/moira#473

### CODE REVIEW

**Comment by beevee** [comment]
Wait for it, I didn't implement the counter yet :-)

**Comment by beevee** [comment]
`StatusIndicator` doesn't currently support 4-status circles. Will open separate issue.

**Comment by sashasushko** [comment]
> StatusIndicator doesn't currently support 4-status circles. Will open separate issue.

@beevee it's not bug, it's feature. DELETE, NODATA and other important statuses always fill all indicator

**Comment by beevee** [comment]
> DELETE, NODATA and other important statuses always fill all indicator

No, they don't. At least, it is not a part of `StatusIndicator` behavior. When I render `StatusIndicator` with 4+ statuses, I get an empty picture. It is not filled with important statuses.

Obviously, some external code does the important status filtering. So we can make `StatusIndicator` support 4 statuses without breaking this external code, and preserve old behavior in all other places, where we don't need 4. Win-win.

**Comment by sashasushko** [comment]
> No, they don't.

 Maybe, but source logic is "max 3 colors"

**Comment by sashasushko** [comment]
> So we can make StatusIndicator support 4 statuses without breaking this external code

Yes, we can. My message is only about "this wasn't bug"

**Comment by beevee** [comment]
Made commit history just a tiny bit cleaner.

File: src/Containers/NotificationListContainer.jsx [hunk 1]
```diff
@@ -49,7 +50,7 @@ class NotificationListContainer extends React.Component<Props, State> {
         return (
             <Layout loading={loading} error={error}>
                 <LayoutContent>
-                    <LayoutTitle>Notifications</LayoutTitle>
+                    <LayoutTitle>Notifications ({list && list.length})</LayoutTitle>
```
Comment (sashasushko) [inline src/Containers/NotificationListContainer.jsx]:
```suggestion
                    <LayoutTitle>Notifications {Array.isArray(list) ? `(${list.length})` : null}</LayoutTitle>
```

Comment (sashasushko) [reply src/Containers/NotificationListContainer.jsx]: Иначе будет `Notifications ()` на пустой странице

Comment (beevee) [reply src/Containers/NotificationListContainer.jsx]: Done!

**Pull request merged**